src/main/scala/com/tuplejump/snackfs/api/model/CreateFileCommand.scala

```diff
@@ -48,27 +55,27 @@ object CreateFileCommand extends Command {
             processId: UUID,
             statistics: Statistics,
             subBlockSize: Long,
-            atMost: FiniteDuration): FSDataOutputStream = {
+            atMost: FiniteDuration,
+            useLocking: Boolean): FSDataOutputStream = {
```

Comment:
Should set this param to true by default. Can result in unexpected behavior if this is false.